### ISSUE TITLE
gnome crashes when using gtk.catppuccin.cursor.enable = true

### ISSUE THREAD

**Issue by bbigras** [catppuccin/nix#216]
see https://github.com/NixOS/nixpkgs/issues/317886 with full log.

```nix
{
  gtk = {
    enable = true;
    gtk2.extraConfig = "gtk-application-prefer-dark-theme = true";
    gtk3.extraConfig.gtk-application-prefer-dark-theme = true;
    catppuccin = {
      enable = true;
      cursor.enable = true;
      gnomeShellTheme = true;
      icon.enable = true;
    };
  };
}
```

also using `catppuccin.flavor = "mocha";`

**Comment by jtojnar** [comment]
https://discourse.nixos.org/t/mouse-cursors-broke-after-recent-nixos-rebuild/46790
Should be fixed by #212

**Comment by getchoo** [comment]
should be solved as of 6336fb8ba1d33498869980ba6b8ce44b25eddf91

feel free to reopen if not :)

**Comment by bbigras** [comment]
It seems fine. Thanks everyone!
